### PR TITLE
Enable console tests

### PR DESCRIPTION
**Pull request by jhou1**
With [DPTP-2532](https://issues.redhat.com/browse/DPTP-2532) ready, reenable the console test cases in Prow. cc @liangxia @yapei

### CODE REVIEW

File: ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml [hunk 1]
```diff
@@ -12,16 +12,17 @@
...
       memory: 4Gi
+      ci-operator.openshift.io/shm: 2g
   env:
   - name: E2E_RUN_TAGS
     default: "@aws-ipi"
     documentation: "Executed cucumber tags"
   - name: E2E_SKIP_TAGS
-    default: "not @console
-          and not @destructive
+    default: "and not @destructive
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml:25]: Extra `and` there.

Comment (jhou1) [reply ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml]: Thanks for catching!

**Pull request closed without merging**